### ISSUE TITLE
comeback?

### ISSUE THREAD

**Issue by WLTN1** [tpatel/fortnite-replay-reader#5]
Hi, will you continue to dev. this project? I would appreciate that :)

**Comment by tpatel** [comment]
Hi @WLTN1, the original idea behind publishing this project was that hopefully a couple of other people would help. I'll try to allocate a bit of time on this project but it might not be enough to keep it up to date.

**Comment by WLTN1** [comment]
Thank you@tpatel ! I have two questions :)
1. Is it possible event "playerelim split into kill and knock? (in duos / squads)
2. Is it possible to find out who the duo / squad team player was?

**Comment by TheWebs** [comment]
> Thank you@tpatel ! I have two questions :)
> 
> 1. Is it possible event "playerelim split into kill and knock? (in duos / squads)
> 2. Is it possible to find out who the duo / squad team player was?

If i may:
1 & 2 -  it sounds very possible as the replay system separates the squads by numbers and knows when u got knocked down

**Comment by TheWebs** [comment]
ps: i dont know whats ur goal but i once made a replay fixer however due to guns removal the game crashed whenever a removed weapon was trying to get loaded

**Comment by julianmacagno** [comment]
Hi, first of all sorry for my bad english.  I discovered your repo a cupple days ago and I really liked it; so much that I decided i'd like to contribut at you if you want. I developed a tiny web page and a javascript script, that uses the vim.js library to draw a tree with the kills you extract from the chunks data. It's beauty to see the graph. also I can provide a bunch of replay files to test it them. Tell me please if you are interested and how we can merge the code. Regards, julianmacagno.
![descarga](https://user-images.githubusercontent.com/20914201/57043466-f7efbf00-6c3d-11e9-9fc1-78814fae62ba.png)

**Comment by julianmacagno** [comment]
Hi, first of all sorry for my bad english.  I discovered your repo a cupple days ago and I really liked it; so much that I decided i'd like to contribut at you if you want. I developed a tiny web page and a javascript script, that uses the vim.js library to draw a tree with the kills you extract from the chunks data. It's beauty to see the graph. also I can provide a bunch of replay files to test it them. Tell me please if you are interested and how we can merge the code.

I attached below a few pictures of my job. Is not in Tree style yet, but is in Graph style. It will not require lot of job; it's only that at the moment I posted this, I didn't.

![descarga](https://user-images.githubusercontent.com/20914201/57044825-3ab39600-6c42-11e9-90f6-8f1a96667cf5.png)
![descarga (1)](https://user-images.githubusercontent.com/20914201/57044826-3ab39600-6c42-11e9-821a-0cefb31f5798.png)

[test.zip](https://github.com/tpatel/fortnite-replay-reader/files/3136119/test.zip)

I attached too my code to you can see it. To run it, first do node index.js (it is a little bit modified to get the exact result i expected); due to the fact I can't load files in Google Chrome the same you do with node, I had to copy the output of the previous command and paste in my code as an array like you can see in /scripts/script.js

Then open index.html in your browser to see the graph, or entry at https://julianmacagno.github.io/.

Hope you like it. Regards, julianmacagno.

**Comment by TheWebs** [comment]
That seems like a cool project, maybe i can help. If you're interested

**Comment by julianmacagno** [comment]
Of course, all help is welcome. My plan is continue developing this script, ordering the data in a Tree with the winner on the top level and their kills under them. Also I want to calculate player's position and improve the algorithm for more performance and to distinct between kills and knocks. I'd probably wait a few days to continue, because I have to study now haha.

**Comment by julianmacagno** [comment]
Ey, go https://github.com/julianmacagno/julianmacagno.github.io. I updated my code, now it draw kills in Tree style. Here you can see the result online https://julianmacagno.github.io/. Hope you like it

**Comment by jokoast** [comment]
@TheWebs @julianmacagno do you know where is the unique game_id in the replay ? 
I'm desperatly trying to find it and i don't.

**Comment by julianmacagno** [comment]
I don't. In fact I ignored that this game_id existed

**Comment by jokoast** [comment]
This "gameID" was used for months when we used to play snipes games. To be sure we were in the same game we copied the 4 last digits of this ID. (same way, all epic account have their own userID [now used in demos to replace string names])
